src/tests/unit/tests/common/components/copy-issue-details-button.test.tsx

```diff
@@ -28,15 +37,35 @@ describe('CopyIssueDetailsButtonTest', () => {
 
     test('render', () => {
         const result = Enzyme.shallow(<CopyIssueDetailsButton {...props} />);
-        expect(result.getElement()).toMatchSnapshot();
+        expect(result.debug()).toMatchSnapshot();
     });
 
-    test('render after click shows toast', () => {
-        const result = Enzyme.shallow(<CopyIssueDetailsButton {...props} />);
+    test('render after click shows toast', async () => {
+        navigatorUtilsMock
+            .setup(navigatorUtils => navigatorUtils.copyToClipboard(issueDetailsText))
+            .returns(() => {
+                return Promise.resolve();
+            })
+            .verifiable(Times.once());
+
+        const result = Enzyme.mount(<CopyIssueDetailsButton {...props} />);
         const button = result.find(DefaultButton);
         onClickMock.setup(m => m(It.isAny())).verifiable(Times.once());
-        button.simulate('click');
-        expect(result.getElement()).toMatchSnapshot();
-        onClickMock.verifyAll();
+        // tslint:disable-next-line: await-promise
+        await button.simulate('click');
+
+        const toast = result.find(Toast);
+
+        expect(toast.state().toastVisible).toBe(true);
+        expect(toast.state().content).toBe('Failure details copied.');
+        expect(result.debug()).toMatchSnapshot();
```

Comment:
I think maybe it would be better to snapshot the `toast.props()` rather than `result.debug()` - that would avoid this unit test's snapshot depending on the implementation details of the render methods of `Toast` and `DefaultButton` (they have their own unit tests for that :) )

Reply:
- if I use the props to snapshot the toast, the snapshot result isn't that informative for example:
`const toast = result.find(Toast);
const toastResult = Enzyme.mount(<Toast {...toast.props()} />);
expect(toastResult.debug()).toMatchSnapshot();`
the result is 
`exports[CopyIssueDetailsButtonTest render after click shows toast 2] = "<Toast deps={{...}} timeoutLength={6000} />";
- We use debug instead of getElement because getElement  doesn't work while it's expected to do, we do it in other files like start-over-drop-down.test I think
`